bitcoin/src/address/script_pubkey.rs

```diff
@@ -48,7 +48,11 @@ define_extension_trait! {
 
         /// Computes P2TR output with a given internal key and a single script spending path equal to
         /// the current script, assuming that the script is a Tapscript.
-        fn to_p2tr<C: Verification>(&self, secp: &Secp256k1<C>, internal_key: UntweakedPublicKey) -> ScriptBuf {
+        fn to_p2tr<C: Verification>(
+            &self,
+            secp: &Secp256k1<C>,
+            internal_key: UntweakedPublicKey,
```

Comment:
Note regarding commit message: it's about the trailing comma, not the lines. White space doesn't matter.

But whatever, it's not important.

Reply:
You are correct, `rustfmt` adds the trailing comma when moving function parameters to new lines. Thats what made me hit this issue.